### PR TITLE
Improve stacktrace environment output for easier copying.

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
PTAL when you have time. This is a reland of an old CL to make it easier to copy environment variables.

Note that this should not be merged until https://github.com/google/clusterfuzz-tools/pull/549 has been deployed.

### CODE REVIEW

**Comment by mbarbella-chromium** [comment]
Just realized this needs a few more changes on the tools side. I'll prepare another change there before landing this.

File: src/python/system/environment.py [hunk 1]
```diff
@@ -319,8 +324,12 @@ def get_environment_settings_as_string():
       if not environment_variable_value:
         continue
 
-      environment_string += '[Environment] %s = %s\n' % (
-          environment_variable, environment_variable_value)
+      if platform() == 'WINDOWS':
+        environment_string += '[Environment] set "%s=%s"\n' % (
+            quote(environment_variable), quote(environment_variable_value))
+      else:
+        environment_string += '[Environment] export %s="%s"\n' % (
```
Comment (inferno-chromium) [inline src/python/system/environment.py]: Do we really need export and set as example here ? We can skip those i think?

File: src/python/system/environment.py [hunk 2]
```diff
@@ -319,8 +324,12 @@ def get_environment_settings_as_string():
       if not environment_variable_value:
         continue
 
-      environment_string += '[Environment] %s = %s\n' % (
-          environment_variable, environment_variable_value)
+      if platform() == 'WINDOWS':
+        environment_string += '[Environment] set "%s=%s"\n' % (
```
Comment (inferno-chromium) [inline src/python/system/environment.py]: i don't think quotes around %s=%s work, it should %s="%s"

File: src/python/system/environment.py [hunk 3]
```diff
@@ -308,7 +313,7 @@ def get_environment_settings_as_string():
       sanitizer_options_file_path = (
           android.device.get_sanitizer_options_file_path('ASAN'))
       environment_string += (
-          '[Environment] ASAN Options file = %s with contents %s \n' %
+          '[Environment] ASAN options file "%s" with contents %s \n' %
```
Comment (inferno-chromium) [inline src/python/system/environment.py]: change "contents %s \n" to "contents:\n%s\n" to match others.

**Comment by inferno-chromium** [comment]
clusterfuzz-tools side might have to strip quotes ?

File: src/python/system/environment.py (hunk 1 above)
Comment (mbarbella-chromium) [reply src/python/system/environment.py]: Leaving them out SGTM.

File: src/python/system/environment.py (hunk 2 above)
Comment (mbarbella-chromium) [reply src/python/system/environment.py]: Shouldn't matter after the change to remove export/set.

**Comment by mbarbella-chromium** [comment]
> clusterfuzz-tools side might have to strip quotes ?

Right. I'll prepare that change as well before landing this and releasing the newer version.

File: src/python/system/environment.py (hunk 3 above)
Comment (mbarbella-chromium) [reply src/python/system/environment.py]: Done

File: src/python/system/environment.py (hunk 1 above)
Comment (mbarbella-chromium) [reply src/python/system/environment.py]: Done

**Pull request merged**